index.html

```diff
@@ -227,7 +228,7 @@ <h1 class="edith-title">Multi-Post Stories</h1>
             <li class="plang3"><p>javascript</p></li>
           </ul>
           <div class="action ediths-tonic">
-            <button id="seeProjBtn2" class="see-project-btn  ediths-tonic go-ediths" onclick="popupDetails(4)">see projects</button>
+            <button id="seeProjBtn4" class="see-project-btn  ediths-tonic go-ediths" onclick="popupDetails(4)">see projects</button>
```

Comment:
- [ optional] since the previous reviewer didn't make note of it, I am putting it as optional. Please kindly ensure that when the popup comes up it should not be possible to manipulate your portfolio 
- [ optional] kindly ensure that the popup menu is full height or the user should not scroll before seeing the rest of it.